repo_health/check_github.py

```diff
@@ -168,7 +182,13 @@ async def check_settings(all_results, github_repo):
     results["last_push"] = github_repo.pushed_at
     repo_license = github_repo.license
     if repo_license is None:
-        results["license"] = None
+        if (
+            github_repo.name in repo_license_exemptions
+            and github_repo.owner.login == repo_license_exemptions[github_repo.name]["owner"]
```

Comment:
I believe this can be shortened to this get-chain:
```suggestion
            github_repo.owner.login == repo_license_exemptions.get(github_repo.name, {}).get("owner")
```
...but only if you want to; it's fine as is.